src/entities/context.ts

```diff
@@ -0,0 +1,12 @@
+export interface Context {
+  dct: string;
+  edc: string;
+  dcat: string;
+  odrl: string;
+  dspace: string;
+}
+
+export interface ContextProperties {
+  "@context": Partial<Context>;
```

Comment:
Can a `@context` have more than one field? 

E.g.
```ts
const a: Context = {
  dct: "...",
  edc: "...",
}
```

If not, I don't think `Partial` is the right generic type.

Reply:
Now I see the response type. Forget my Q.